npps4/webui/endpoints/unlock_backgrounds.py

```diff
@@ -0,0 +1,60 @@
+import fastapi
+
+from .. import template
+from ...app import webui
+
+import sqlalchemy
```

Comment:
`import sqlalchemy` should be alongside `import fastapi` in alphabetical order. This means `import sqlalchemy` must be below `import fastapi`.